src/App.tsx

```diff
@@ -1,9 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  // const filterMovies = (searchKey: string) => {
+  //   const input = searchKey.toLowerCase().trim();
+
+  //   return moviesFromServer.filter(movie => {
+  //     const title = movie.title.toLowerCase();
+  //     const description = movie.description.toLowerCase();
+
+  //     return title.includes(input) || description.includes(input);
+  //   });
+  // };
+
+  const filterMovies = (searchWord: string) => {
+    const word = searchWord.toLowerCase().trim();
+
+    return moviesFromServer.filter(movies => {
+      const title = movies.title.toLowerCase();
+      const description = movies.description.toLowerCase();
```

Comment:
```suggestion
    return moviesFromServer.filter(movie => {
      const title = movie.title.toLowerCase();
      const description = movie.description.toLowerCase();
```
because filter method checks every movie one by one in the list
movie would be a more logical name

Reply:
I got it Nurdan thanks